modules/tftp/server.rb

```diff
@@ -63,6 +63,23 @@ def delete_file(file)
         logger.debug "TFTP: Skipping a request to delete a file which doesn't exists"
       end
     end
+
+    def delete_host_dir(mac)
+      host_dir = "#{path}/host_config/#{dashed_mac(mac).downcase}"
+      if Dir.exist?(host_dir)
+        FileUtils.rm_rf host_dir
+        logger.debug "TFTP: #{host_dir} removed successfully"
+      else
+        logger.debug "TFTP: Skipping a request to delete a directory which doesn't exists"
+      end
+    end
+
+    def setup_bootloader(mac, os, major, minor, arch, bootfilename_efi, build)
```

Comment:
This is a lot of parameters. Would it make sense to make them named parameters?
```suggestion
    def setup_bootloader(mac:, os:, major:, minor:, arch:, bootfilename_efi:, build:)
```

It may also be helpful to use [yardoc](https://yardoc.org/) parameters to document them:

```suggestion
    # @param build [Boolean] Whether the host is in build mode or not
    def setup_bootloader(mac:, os:, major:, minor:, arch:, bootfilename_efi:, build:)
```